packages/extensions/openapi-to-typespec/package.json

```diff
@@ -42,16 +42,16 @@
     "@azure-tools/codegen": "~2.10.0",
     "@autorest/extension-base": "~3.6.0",
     "@autorest/codemodel": "~4.20.0",
-    "@typespec/compiler": "^0.53.0",
-    "@typespec/rest": "^0.53.0",
-    "@typespec/http": "^0.53.0",
-    "@typespec/versioning": "^0.53.0",
-    "@typespec/prettier-plugin-typespec": "^0.53.0",
-    "@azure-tools/typespec-azure-core": "^0.39.0",
-    "@azure-tools/typespec-autorest": "^0.39.0",
-    "@azure-tools/typespec-azure-resource-manager": "^0.39.0",
-    "@typespec/openapi": "^0.53.0",
-    "@typespec/openapi3": "^0.53.0",
+    "@typespec/compiler": "^0.56.0",
+    "@typespec/rest": "^0.56.0",
+    "@typespec/http": "^0.56.0",
+    "@typespec/versioning": "^0.56.0",
+    "@typespec/prettier-plugin-typespec": "^0.56.0",
+    "@azure-tools/typespec-azure-core": "^0.42.0",
+    "@azure-tools/typespec-autorest": "^0.42.0",
+    "@azure-tools/typespec-azure-resource-manager": "^0.42.0",
```

Comment:
can we upgrade to `0.42.1` to see if `ManagedServiceIdentities` issue has been resolved?

Reply:
We are not impacted by that issue, if you still remember.